specification/purview/Azure.Analytics.Purview.DataMap/tspconfig.yaml

```diff
@@ -0,0 +1,36 @@
+parameters:  
+  "service-dir":
+    default: "sdk/azure.analytics.purview.datamap"
```

Comment:
Are we going to use the `azure.analytics.purview.datamap` as folder name? this may not apply JS naming convention in SDK repo and also failed our automation validation. Usually it would be `azureanalyticspurviewdatamap`.

Reply:
should be `sdk/purview`

see https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md#sample-project

Reply:
Thanks. Updated